test/integration/good/compiler-optimizations/mem_patterns/cpp.expected

```diff
@@ -4636,7 +4636,7 @@ class indexing_model final : public model_base_crtp<indexing_model> {
                                  stan::math::multiply(
                                    p_aos_mat_pass_func_outer_single_indexed1,
                                    p_aos_mat_pass_func_outer_single_indexed2)),
-                               "transpose(multiply(p_aos_mat_pass_func_outer_single_indexed1,\np_aos_mat_pass_func_outer_single_indexed2))",
+                               "transpose(multiply(p_aos_mat_pass_func_outer_single_indexed1,\n                   p_aos_mat_pass_func_outer_single_indexed2))",
```

Comment:
Huge space?

Reply:
That is so it is actually well-aligned when printed, e.g.

```
transpose(multiply(p_aos_mat_pass_func_outer_single_indexed1,
                   p_aos_mat_pass_func_outer_single_indexed2))

```